src/Surfnet/ServiceProviderDashboard/Application/Service/MailService.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * Copyright 2022 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Application\Service;
+
+use Exception;
+use Surfnet\ServiceProviderDashboard\Domain\Entity\ManageEntity;
+use Surfnet\ServiceProviderDashboard\Domain\Mailer\Mailer;
+use Surfnet\ServiceProviderDashboard\Infrastructure\DashboardBundle\Factory\MailMessageFactory;
+
+class MailService
+{
+    /** @var MailMessageFactory */
+    private $mailFactory;
+    /** @var Mailer */
+    private $mailer;
+
+    public function __construct(MailMessageFactory $mailFactory, Mailer $mailer)
+    {
+        $this->mailer = $mailer;
+        $this->mailFactory = $mailFactory;
+    }
+
+    public function sendErrorReport(ManageEntity $entity, Exception $exception)
+    {
+        // Inform the service desk of the unavailability of Jira
```

Comment:
Is the unavailability the only case or could there be several reasons? The comment seems unnecessary

Reply:
At this point, we only send an error report when the Jira API was down or otherwise unavailable. I extracted this code from the CommandHandler, so you might be right in your observation that the comment has become obsolete. 

When I'm back on my branch, I'll be sure to check that class with this in mind.